### ISSUE TITLE
Interest in a short paper?

### ISSUE THREAD

**Issue by jradavenport** [jradavenport/GALEX_Boyajian#3]
Determine if there is

1. enough interest in writing a short paper on the short- and long-timescale observations from GALEX
2. enough meat in the plots/data presented to actually accomplish such a paper...

cc: @tsboyajian, @ZacheryLaycock, @daveonebillion, @benmontet, **others?**

**Comment by jradavenport** [comment]
For clarity: I am inviting co-authorship for anyone willing to help me vet, debunk, clarify, plot, write, re-consider, etc....

**Comment by scfleming** [comment]
If I had to bet, I would bet on being able to disprove the 80-second signal.  The longer-term trend looks far more insteresting to me (using the CAUSE data).

**Comment by benmontet** [comment]
Happy to be involved, my interest is unsurprisingly primarily in the long-term trend, but not immediately sure what my contribution would be---I have zero experience with GALEX data.

**Comment by cmillion** [comment]
I agree with the above. I haven't seen anything in the sub-visit photometry (where gPhoton uniquely contributes) that I think is compelling. The long term trend looks really interesting to me, though. The test in #1 would convince me that it was probably real.

**Comment by jradavenport** [comment]
I too agree that the sub-visit data isn't particularly compelling. I'm wondering what other comparisons I could run besides dust models that take advantage of the wavelength separation between Kepler and GALEX

**Comment by davidjwilson** [comment]
I also agree, I wouldn't believe the 80s signal until we can cross check it with (say) HST timetag data.

**Comment by scfleming** [comment]
If the signals are caused by clumps or a swarm of blackbodies with some size distribution, would you expect differences based upon a combination of that size distribution + an albedo, given the flux they receive from the host star at their presumed distance from the star?  This is somehwat similar to the dust I suppose, but conjecturing they are larger bodies that emit as opposed to grains?  In this case, the constraint would be on some sort of effective temperature distribution for some distribution of object size?  I'm thinking of how one can treat this essentially as an SED/excess problem...

**Comment by tsboyajian** [comment]
Hey everyone - these notifications went to my Yale inbox for some reason...  Just catching up now.  I think this is a great result and would love to see it published.  Is there a plan in place? I dont have the slightest bit of expertise in Galex data tho, so I cant help there.

**Comment by jradavenport** [comment]
Great! I think there's sufficient interest in trying to put a quick letter together. I've got a student looking at WISE as well to see if there's a longer wavelength range we can exploit.

I'll starting outlining said letter this week and will shamelessly put you all on it, and send the outline it around for thoughts

**Comment by jradavenport** [comment]
Tinkering with the observed Kepler (~2%) and NUV (~3.5%) drop over these 2 GALEX observations. If we fit them with dust models, I find you need an R_V = 5 to get a "flat" enough UV response and to fit both bands simultaneously.

**Comment by jradavenport** [comment]
FYI an outline of a paper was just put up. It's very rough, but gives idea of where I'm going with this. I want to be generous with authorship since all you wonderful people did so much to help validate this idea! Let me know who I should include! Email me, post here, tweet, whatever, to make comments/suggestions!
